testing/local-amp-chrome-extension/background.js

```diff
@@ -33,7 +37,7 @@ chrome.webRequest.onBeforeRequest.addListener(
       path = path.replace(/\.js$/, '.max.js');
     }
     return {
-      redirectUrl: 'http://localhost:8000/' + path
+      redirectUrl: ( baseUrl || defaultBaseUrl ) + path
```

Comment:
Look at the comment below on how to handle the case when the input is deleted, this will no longer be necessary.